shared/tracker/index.js

```diff
@@ -38,10 +38,65 @@ export type TrackerProps = {
   currentlyFollowing: boolean,
   lastAction: ?('followed' | 'refollowed' | 'unfollowed' | 'error'),
   name?: string,
-  inviteLink?: string,
+  inviteLink?: ?string,
   isPrivate?: boolean
 }
 
+export function trackerPropsToRenderProps (props: TrackerProps): RenderProps {
```

Comment:
This is fine :+1: but I have always disliked this pattern. If the render wants to split things up it can but the smart component should know about this. Different ticket though